src/db/tables/attributions.ts

```diff
@@ -60,7 +60,10 @@ async function addAttributionForSubscriber(
     res = await knex("attributions").insert(attribution).returning("*");
   } catch (e) {
     if ((e as Error).message.includes("violates unique constraint")) {
-      logger.info("Ignoring unique constraints");
+      logger.info("addAttributionForSubscriber", {
```

Comment:
I've been trying to use the style `add_attribution_for_subscriber` for these so they are more like events

Reply:
I'd like to refactor this together if we want to.. The rationale behind using the correct format is that `addAttributionForSubscriber` represents the current function, making it easier to look for in the logs. I think we can also throw in some magic to the `logger` code so it can autodetect the caller function's name and potentially change the name from camel to snake

Reply:
Created: https://mozilla-hub.atlassian.net/browse/MNTOR-2915